bin/setup

```diff
@@ -51,6 +42,15 @@ else
   printf 'Ask for access to the "upcase-production" Heroku app\n'
 fi
 
+# Set up deploys
+if ! git remote | grep 'staging' > /dev/null; then
+  heroku git:remote -a upcase-staging -r staging
```

Comment:
we used to hardcode the heroku git url, but it looks like at some point they changed the format, so when you tried to deploy with remotes created by this script, it would just hang and eventually timeout. I fixed that by just having the heroku CLI setup remotes.